scripts/run_integration_tests_aarch64.sh

```diff
@@ -8,9 +8,73 @@ export BUILD_TARGET=${BUILD_TARGET-aarch64-unknown-linux-gnu}
 
 WORKLOADS_DIR="$HOME/workloads"
 WORKLOADS_LOCK="$WORKLOADS_DIR/integration_test.lock"
+EDK2_BUILD_DIR="$WORKLOADS_DIR/edk2_build"
 
 mkdir -p "$WORKLOADS_DIR"
 
+build_edk2() {
+    EDK2_REPO="https://github.com/jongwu/edk2.git"
```

Comment:
This should eventually point to either somewhere belongs to the CLH community (e.g. a forked branch) or the merged upstream repository